woocommerce/woocommerce-android
# Add toolbar for AddOrderShipmentTrackingFragment and AddOrderTrackingProviderListFragment

Part of #10570 

### Description

This PR introduces individual toolbars for AddOrderShipmentTrackingFragment and AddOrderTrackingProviderListFragment, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

This PR also has a 2 small change
1. Make the order detail view scrollable even from the margins. Read this comment for more details. pdfdoF-4C0-p2#comment-5813
2. Add a small delay before opening the first order on tablets so that it works even when the user logs into the app and visits order listing screen

### Testing instructions

Navigate to the Order detail for a paid order and click on Add tracking CTA. Ensure the toolbar and its action work as expected on both phone and tablet.

=====

Ensure you can scroll order detail screen even from the margins.

=====

Log out and log in back, visit order listing screen on tablet and ensure the first order is selected by default.

### Images/gif

<img width="502" alt="Screenshot 2024-02-13 at 12 29 38 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/8630ceb6-4fcf-465e-a052-f00e7e3dc9b7">
<img width="508" alt="Screenshot 2024-02-13 at 12 29 45 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/b8c2bef8-fb79-4a11-a547-0af60c0adcc7">
